### PR TITLE
OP-438 New HTTP Client for SMS Gateway

### PR DESCRIPTION
**Pull request by AndreiDodu**
configuration (for core project)

### CODE REVIEW

File: src/main/java/org/isf/utils/image/ImageUtil.java [hunk 1]
```diff
@@ -61,10 +63,35 @@ public static BufferedImage scaleImage(final BufferedImage src,
...
+    /**
+     * Converts a given Image into a BufferedImage
+     *
+     * @param img The Image to be converted
+     * @return The converted BufferedImage
+     */
+    public static BufferedImage toBufferedImage(Image img)
+    {
+        if (img instanceof BufferedImage)
+        {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/image/ImageUtil.java:80]: Typically (and in the other examples in this PR) the brace is on the same line as the **if**.  Doing so matches the project's formatting standards.  Thanks

File: src/main/java/org/isf/utils/image/ImageUtil.java [hunk 2]
```diff
@@ -33,11 +34,12 @@
...
 
-    public static BufferedImage scaleImage(final BufferedImage src,
+    public static BufferedImage scaleImage(final Image src,
                                            final int boundWidth,
                                            final int boundHeight){
-        final int originalWidth = src.getWidth();
-        final int originalHeight = src.getHeight();
+    	BufferedImage bi = toBufferedImage(src);
+        final int originalWidth = bi.getWidth();
+        final int originalHeight = bi.getHeight();
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/image/ImageUtil.java:42]: Inconsistent indentation (line 40 vs. lines 41 and 42).

**Comment by mwithi** [comment]
Hi @AndreiDodu, is this PR ready to be published?

File: src/main/java/org/isf/utils/image/ImageUtil.java (hunk 1 above)
Comment (AndreiDodu) [reply src/main/java/org/isf/utils/image/ImageUtil.java:80]: I corrected as you suggested.

File: src/main/java/org/isf/utils/image/ImageUtil.java (hunk 2 above)
Comment (AndreiDodu) [reply src/main/java/org/isf/utils/image/ImageUtil.java:42]: I corrected as you suggested.

**Comment by AndreiDodu** [comment]
> Hi @AndreiDodu, is this PR ready to be published (not draft)?

I committed and made a pull request with new version with some corrections based on your suggestions @mwithi @dbmalkovsky.

**Comment by mwithi** [comment]
Superseded by https://github.com/informatici/openhospital-gui/pull/566

**Pull request closed without merging**